FGEGraphics/UISystem/UIPositionHelper.cs

```diff
@@ -190,10 +190,6 @@ public UIPositionHelper GetterXY(Func<int> x, Func<int> y)
         return this;
     }
 
-    /// <summary>Sets getter X and Y values using a <see cref="Location"/>.</summary>
-    /// <returns>This object.</returns>
-    public UIPositionHelper GetterXY(Func<Location> location) => GetterXY(() => (int) location().X, () => (int) location().Y);
```

Comment:
why is this removed?

Reply:
I don't see a reason for it to exist, was only used in `UIElementText` w.r.t. text alignment implementation. Forgot to remove the constant version

Reply:
It's a nice convenience/utility - remember that FGE is a library, not an exclusive user. ie it very intentionally contains many unused public methods. Library methods should only be removed if there's an active reason to remove them (eg no longer fits design, or encourages bad patterns, or etc).